### PR TITLE
Build cart view

### PR DESCRIPTION
**Pull request by kpearson**

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class OrdersController < ApplicationController
+  def new
+    @items = []
+    @cart.data.each do |item_id, quantity|
+      item = Item.find(item_id.to_i)
+      item.add_quantity(quantity)
+      @items << item
+    end
+  end
+end
```
Comment (bryceholcomb) [inline app/controllers/orders_controller.rb]: I agree with hound

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@ def create
   end
 
   def destroy
-    session.clear
+    session[:user_id] = nil
```
Comment (bryceholcomb) [inline app/controllers/sessions_controller.rb:21]: I like this, no longer clearing the cart within the session too

File: app/models/item.rb [hunk 3]
```diff
@@ -6,4 +6,14 @@ class Item < ActiveRecord::Base
   has_many :categories, through: :category_items
   has_many :line_items
   has_many :orders, through: :line_items
+
+  attr_reader :quantity
+
+  def add_quantity(quantity)
```
Comment (bryceholcomb) [inline app/models/item.rb:12]: Favorite method written today

File: app/views/shared/_items.html.erb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<% @items.each do |item| %>
+  <hr><div class="item" id=<%= item.title %>>
+    <div id="description">
```
Comment (bryceholcomb) [inline app/views/shared/_items.html.erb:3]: @rshea303 this is where you can add the div for the picture within the first "#item" div

File: spec/feature/guest_user_spec.rb [hunk 5]
```diff
@@ -147,7 +99,41 @@
...
+    end
+    within "div#description" do
+      expect(page).to have_content("The classic breakfast dish")
+    end
+    within "div#price" do
+      expect(page).to have_content("$10.00")
+    end
+  end
+
+  def click_add_to_cart_link(category)
```
Comment (kpearson) [inline spec/feature/guest_user_spec.rb:129]: _Helper method to add item to cart.  Removing duplication._

File: spec/feature/guest_user_spec.rb [hunk 6]
```diff
@@ -46,62 +31,36 @@
...
-        click_link "Add to Cart"
-      end
-    end
-    within("div#cart-contents") do
+  it "can add an item to a cart" do
+    click_add_to_cart_link("Breakfast")
+    within("#cart-contents") do
       expect(page).to have_content("1")
     end
   end
```
Comment (bryceholcomb) [inline spec/feature/guest_user_spec.rb:40]: This helper method cuts down so much code haha

**Pull request merged**